micro-spring-boot/src/test/java/app/rest/client/com/oath/micro/server/RestClientTest.java

```diff
@@ -29,6 +36,7 @@
 import com.oath.micro.server.spring.boot.MicroSpringBoot;
 
 @Microserver @MicroSpringBoot
+@Ignore //not picking up Guava Jackson config
```

Comment:
Will this cause any issues in systems that use this?

Reply:
Yeah, it's a bug that needs fixed.